client/src/components/Flash/index.tsx

```diff
@@ -25,7 +26,7 @@ function Flash({ flashMessage, removeFlashMessage }: FlashProps): JSX.Element {
 
   return (
     <TransitionGroup>
-      <CSSTransition classNames='flash-message' key={id} timeout={500}>
+      <CSSTransition key={id} timeout={500}>
         <Alert variant={flashStyle} className='flash-message'>
           {t(message, variables)}
           <CloseButton
```

Comment:
I don't mind building the dismissbility into the component, I have two requests:

- It doesn't import `react-i18next`
- it uses generic `button` element, not `CloseButton`

Reply:
I didn't mean uses as make new `button`, but the export of `CloseButton` shouldn't be included, it can be used in the alert but not exported to the client because it isn't needed there anymore.